Modules/TPC/src/CheckOfPads.cxx

```diff
@@ -0,0 +1,377 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   CheckOfPads.cxx
+/// \author Laura Serksnyte, Maximilian Horst
+///
+
+#include "TPC/CheckOfPads.h"
+#include "QualityControl/MonitorObject.h"
+#include "QualityControl/Quality.h"
+#include "QualityControl/QcInfoLogger.h"
+
+// ROOT
+#include <TCanvas.h>
+#include <TH1.h>
+#include <TH2.h>
+#include <TPad.h>
+#include <TList.h>
+#include <TPaveText.h>
+
+#include <iostream>
+
+namespace o2::quality_control_modules::tpc
+{
+void CheckOfPads::configure()
+{
+  if (auto param = mCustomParameters.find("mediumQualityPercentageOfWorkingPads"); param != mCustomParameters.end()) {
```

Comment:
Here is a lot of copy'n'paste. And there is already a function for this: https://github.com/AliceO2Group/QualityControl/blob/352b520fbc9187ed7e857e263946d3c6b0580ce6/Modules/Common/include/Common/Utils.h
Used e.g. here: https://github.com/AliceO2Group/QualityControl/blob/master/Modules/TPC/src/PID.cxx#L46
Perhaps you can patch `Common/Utils.h` with
```diff
diff --git a/Modules/Common/include/Common/Utils.h b/Modules/Common/include/Common/Utils.h
index ad0127ca..15b14e8b 100644
--- a/Modules/Common/include/Common/Utils.h
+++ b/Modules/Common/include/Common/Utils.h
@@ -30,15 +30,14 @@ namespace o2::quality_control_modules::common
 /// \param name Name of the taskParameter
 /// \return taskParameter converted to bool, int, float, double or std::string depending on the template type
 template <typename T>
-T getFromConfig(const std::unordered_map<std::string, std::string>& params, const std::string_view name)
+T getFromConfig(const std::unordered_map<std::string, std::string>& params, const std::string_view name, T retVal = T{})
 {
   const auto last = params.end();
   const auto itParam = params.find(name.data());
-  T retVal{};
 
   if (itParam == last) {
     LOGP(warning, "missing parameter {}", name.data());
-    LOGP(warning, "Please add '{}': '<value>' to the 'taskParameters'.", name.data());
+    LOGP(warning, "Please add '{}': '<value>' to the 'taskParameters'. Using default value {}.", name.data(), retVal);
   } else {
     const auto& param = itParam->second;
     if constexpr (std::is_same<int, T>::value) {
```
and update all occurrances below with something like:
```c++
mMediumQualityLimit = common::getFromConfig<float>(mCustomParameters, "mediumQualityPercentageOfWorkingPads", 0.7);
```